core/src/avm2/e4x.rs

```diff
@@ -239,10 +239,14 @@ impl<'gc> E4XNode<'gc> {
         let mut parser = Reader::from_str(&data_utf8);
         let mut open_tags: Vec<E4XNode<'gc>> = vec![];
 
+        let namespace = activation.avm2().public_namespace;
+        let xml_class = activation.resolve_class(&Multiname::new(namespace, "XML")).unwrap();
+
         // FIXME - look these up from static property and settings
         let ignore_comments = true;
         let ignore_processing_instructions = true;
-        let ignore_white = true;
+
+        let ignore_white = xml_class.get_public_property("ignoreWhitespace", activation).unwrap_or(Value::Bool(true)).coerce_to_boolean();
```

Comment:
This isn't quite right - the value should be read when an `XML` object is constructed, and stored in the `settings` object. For now, I think it's good enough to keep hardcoding `ignore_white = true`, and check it before trimming.